### PR TITLE
not assert task.load_data_frame(required_columns={})

### PR DESCRIPTION
**Pull request by vaaaaanquish**
#179

### CODE REVIEW

**Comment by vaaaaanquish** [comment]
@Hi-king @hirosassa please check PR:)

File: test/test_task_on_kart.py [hunk 1]
```diff
@@ -351,9 +351,6 @@ def test_load_index_only_dataframe(self):
         task = _DummyTask()
         task.load = MagicMock(return_value=pd.DataFrame(index=range(3)))
 
-        # connnot load index only frame with required_columns
-        self.assertRaises(AssertionError, lambda: task.load_data_frame(required_columns={'a', 'c'}))
```
Comment (vaaaaanquish) [inline test/test_task_on_kart.py:355]: A curious assertion added here.
https://github.com/m3dev/gokart/pull/137

File: test/test_task_on_kart.py [hunk 2]
```diff
@@ -314,7 +314,7 @@ def test_add_cofigureation_evaluation_order(self, mock_cmdline: MagicMock):
         class DummyTaskAddConfiguration(gokart.TaskOnKart):
             aa = luigi.IntParameter()
 
-        luigi.configuration.get_config().set(f'DummyTaskAddConfiguration', 'aa', '3')
+        luigi.configuration.get_config().set('DummyTaskAddConfiguration', 'aa', '3')
```
Comment (vaaaaanquish) [inline test/test_task_on_kart.py:317]: f-string is missing placeholders [F541]

Comment (Hi-king) [reply test/test_task_on_kart.py:317]: 🆒

File: test/test_task_on_kart.py [hunk 3]
```diff
@@ -329,6 +329,14 @@ def test_load_list_of_list_pandas(self):
         self.assertIsInstance(df, pd.DataFrame)
         self.assertEqual(3, df.shape[0])
 
+    def test_load_dict_of_pandas(self):
```
Comment (Hi-king) [inline test/test_task_on_kart.py:332]:
```suggestion
    def test_load_single_value_dict_of_dataframe(self):
```

**Comment by Hi-king** [comment]
LGTM!

**Review by Hi-king — APPROVED** [review]

File: test/test_task_on_kart.py (hunk 2 above)
Comment (hirosassa) [reply test/test_task_on_kart.py:317]: 👍

**Review by hirosassa — APPROVED** [review]
LGTM

**Comment by Hi-king** [comment]
@vaaaaanquish Merged. Thx for your contribution!

**Pull request merged**